### ISSUE TITLE
amiusingrequestly http header

### ISSUE THREAD

**Issue by foolioo** [requestly/requestly-desktop-app#22]
## Summary

[Requestly adds `amiusingrequestly` HTTP header to requests](https://github.com/requestly/requestly-proxy/blob/658fc1c1f29bd0b4966a260325132f607e309211/src/components/proxy-middleware/middlewares/amiusing_middleware.js#L12). How can I make Requestly not to send the `amiusingrequestly` header?

**Comment by sachinjain024** [comment]
Thanks! for creating this ticket! I created a duplicate one here - https://github.com/requestly/requestly/issues/285

This is a harmless header which we added earlier for debugging purposes. Curious to know if this impacts you in a certain way. Irrespective we will fix this issue in the next release.

PS - We track all the issues in the parent repository so I will tag this ticket there as well. Request you to watch this issue - https://github.com/requestly/requestly/issues/285 for future updates.
